nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -44,4 +44,6 @@
 
 - A new option was added to the virtualisation module that enables specifying explicitly named network interfaces in QEMU VMs. The existing `virtualisation.vlans` is still supported for cases where the name of the network interface is irrelevant.
 
+- DocBook option documentation is no longer supported, all module documentation must now use markdown.
```

Comment:
```suggestion
- DocBook option documentation is no longer supported, all module documentation now uses markdown.
```

Otherwise this sounds negative, like another restriction. But actually this is a good thing.